Clinical-Genomics/scout
# Populate clinical assessments for case pinned, causative and partial c…

Some clinical assessments are not currently shown on pinned and causative variants. This PR fixes this behavior by **populating clinical assessment for all pinned, causative, partial causative and evaluated variants** of a case.
Fix #2710 

**How to test**:
1. On a local instance of Scout, master branch, pin a variant and then dismiss it.
2. Notice that on variantS page, the variant is dismissed:
![image](https://user-images.githubusercontent.com/28093618/121864997-c3e09680-ccfd-11eb-9106-bb0d657ead5d.png)
3. On case page, the same pinned variant doesn't show the "dismissed" badge.
![image](https://user-images.githubusercontent.com/28093618/121865382-2b96e180-ccfe-11eb-8caa-38cbb5ec4a75.png)
4. Switch to this branch and make sure that the badge is instead present on case page
5. Other tests: try to add ACMG classification, manual rank and mosaic tag to the variant. Also these badges should show on case page.


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
